portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IdentityEvents.testcase

```diff
@@ -974,4 +1136,117 @@ definition {
+		property analytics.cloud.upstream = "false";
+		property proxy.server.enabled = "true";
+
+		var usernameList = "user1,user2";
+
+		task ("Add two new users") {
+			for (var username : list "${usernameList}") {
+				JSONUser.addUser(
+					userEmailAddress = "${username}@liferay.com",
+					userFirstName = "${username}",
+					userLastName = "${username}",
+					userScreenName = "${username}");
+
+				JSONUser.setFirstPassword(
+					agreeToTermsAndAnswerReminderQuery = "true",
+					requireReset = "false",
+					userEmailAddress = "${username}@liferay.com");
+			}
+		}
+
+		task ("Sync the Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Start Har recording") {
+			ProxyUtil.startHarRecording("identity");
+		}
+
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "user1@liferay.com",
+				userLoginFullName = "user1 user1");
+		}
+
+		task ("Visit the document") {
+			ACUtils.navigateToSitePage(
+				actionType = "View DM",
+				documentTitleList = "DM AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Wait for sending requests") {
+			ACUtils.waitForSendingRequest();
+		}
+
+		task ("Get the emailAddressHashed number of the identity event") {
+			var emailAddressHashed1 = ACUtils.getPageRefRequestPropertyValue(
+				pageRef = "identity",
+				property = "emailAddressHashed");
+		}
+
+		task ("Check that the emailAddressHashed is not empty") {
+			TestUtils.assertNotEquals(
+				actual = "${emailAddressHashed1}",
+				expected = "");
+		}
+
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "user2@liferay.com",
+				userLoginFullName = "user2 user2");
+		}
+
+		task ("Download the document") {
+			ACUtils.navigateToSitePage(
+				actionType = "Download DM",
+				documentTitleList = "DM AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Wait for sending requests") {
+			ACUtils.waitForSendingRequest();
+		}
+
+		task ("Get the emailAddressHashed number of the identity event") {
+			var emailAddressHashed2 = ACUtils.getPageRefRequestPropertyValue(
+				pageRef = "identity",
+				property = "emailAddressHashed");
+		}
+
+		task ("Check if the emailAddressHashed was modified") {
+			TestUtils.assertNotEquals(
+				actual = "${emailAddressHashed2}",
+				expected = "${emailAddressHashed1}");
+		}
+
+		task ("Close sessions") {
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("Switch the property in AC") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${assignedPropertyName}");
+		}
+
+		task ("Navigate to the Known Individuals on the Individuals dashboard") {
+			ACNavigation.goToIndividuals();
+
+			ACIndividualsDashboard.goToKnownIndividualsTab();
+		}
+
+		task ("Check that the two individuals are classified as known individuals") {
+			ACUtils.viewNameList(nameList = "${usernameList} ${usernameList}");
+		}
+	}
```

Comment:
Hey Vini, regarding the changes that Marcellus talked about I think you just need to change `NewUserChangeEmailAddressHashed`, since you are using two users.